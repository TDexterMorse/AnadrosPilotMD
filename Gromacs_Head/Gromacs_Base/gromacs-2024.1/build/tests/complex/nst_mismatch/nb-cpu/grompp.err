                      :-) GROMACS - gmx grompp, 2024.1 (-:

Executable:   /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/bin/gmx
Data prefix:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1 (source tree)
Working dir:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/complex/nst_mismatch/nb-cpu
Command line:
  gmx grompp -f /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/nst_mismatch/grompp.mdp -c /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/nst_mismatch/conf -r /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/nst_mismatch/conf -p /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/nst_mismatch/topol -maxwarn 10

Ignoring obsolete mdp entry 'ns_type'

WARNING 1 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/nst_mismatch/grompp.mdp]:
  The Berendsen barostat does not generate any strictly correct ensemble,
  and should not be used for new production simulations (in our opinion).
  We recommend using the C-rescale barostat instead.

Generating 1-4 interactions: fudge = 0.5
Number of degrees of freedom in T-Coupling group System is 1293.00

There was 1 WARNING

Thanx for Using GROMACS - Have a Nice Day

