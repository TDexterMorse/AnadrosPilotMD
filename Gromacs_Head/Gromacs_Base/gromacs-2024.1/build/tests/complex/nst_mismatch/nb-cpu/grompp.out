Generated 330891 of the 330891 non-bonded parameter combinations

Generated 330891 of the 330891 1-4 parameter combinations

Excluding 2 bonded neighbours molecule type 'SOL'

turning H bonds into constraints...
Analysing residue names:
There are:   216      Water residues

The largest distance between excluded atoms is 0.165 nm between atom 185 and 186
Calculating fourier grid dimensions for X Y Z
Using a fourier grid of 16x16x16, spacing 0.116 0.116 0.116

Estimate for the relative computational load of the PME mesh part: 0.28

This run will generate roughly 0 Mb of data
